extensions/container-image/runtime/pom.xml

```diff
@@ -25,6 +25,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-mutiny</artifactId>
+        </dependency>
```

Comment:
Do we really want to add a runtime dependency just for the sake of the Dev UI?

Reply:
Well, I need to stream, and there is not much I can do without Mutiny. I don't believe it's a problem, as mutiny will likely be there.